tests/lax_numpy_test.py

```diff
@@ -1752,6 +1752,27 @@ def testTrimZeros(self, a_shape, dtype, trim):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_rank{}".format(
+        jtu.format_shape_dtype_string(a_shape, dtype), rank),
+       "dtype": dtype, "a_shape": a_shape, "rank": rank}
+      for rank in (1, 2)
+      for dtype in default_dtypes
+      for a_shape in one_dim_array_shapes))
+  def testPoly(self, a_shape, dtype, rank):
+    if np.mintypecode(dtype(0).dtype.char) == 'd' and not config.x64_enabled:
+      self.skipTest(f"Only run {dtype} testcase when float64 is enabled.")
```

Comment:
I don't think this check should be needed... `default_dtypes` already skips unavailable types.

Reply:
Clarification: this check should not be needed once the `mintypecode` logic in the function is replaced with `_promote_dtypes_inexact`.

Reply:
Yes, that makes sense. https://github.com/google/jax/commit/79d331db2fa4275f0cea4c53131c087398b9fb56 implements this suggestion.